crates/bevy_window/src/window.rs

```diff
@@ -5,6 +5,26 @@ use raw_window_handle::RawWindowHandle;
 #[derive(Copy, Clone, Debug, PartialEq, Eq, Hash)]
 pub struct WindowId(Uuid);
 
+/// Behavior of the presentation engine based on frame rate.
```

Comment:
We should have a very brief explanation of what a presentation engine is here (in the detailed part of the doc string). It's very jargon heavy and users who aren't focusing on rendering are going to want to play with this setting.

Reply:
Agreed. I originally copied the definition and docs from `wgpu`. I think their enum variant docs are solid but the initial doc is a bit terse. I've updated this to be more descriptive and summarizing the options.